src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Tcp/IdentityTests.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.ServiceModel;
+using System.Text;
+using Xunit;
+
+namespace Security.TransportSecurity.Tests.Tcp
+{
+    public static class IdentityTests
+    {
+        [Fact]
+        [ActiveIssue(12)]
+        [OuterLoop]
+        //Once this feature is enabled, we will need to add negative tests
+        public static void VerifyServiceDNS()
```

Comment:
Agree it's hard to tell what's being tested from the test itself. I will add below comment:
"The product code will check the Dns identity from the server and throw if it does not match what is specified in DnsEndpointIdentity".

I will rename the test to VerifyServiceIdentityMatchDnsEndpointIdentity